diff --git a/dist/js/tools/utils.js b/dist/js/tools/utils.js
index 613e99e..121909f 100644
--- a/dist/js/tools/utils.js
+++ b/dist/js/tools/utils.js
@@ -9,6 +9,12 @@
 catch (e) {
     countLines = require('metro/src/lib/countLines');
 }
+countLines =
+    typeof countLines === 'function'
+        ? countLines
+        : typeof countLines === 'object' && typeof countLines.default === 'function'
+            ? countLines.default
+            : (string) => (string.match(/\r\n?|\n|\u2028|\u2029/g) || []).length + 1;
 /**
  * Returns minified Debug ID code snippet.
  */
diff --git a/scripts/expo-upload-sourcemaps.js b/scripts/expo-upload-sourcemaps.js
index 7003b53..d62c38a 100755
--- a/scripts/expo-upload-sourcemaps.js
+++ b/scripts/expo-upload-sourcemaps.js
@@ -218,7 +218,7 @@ for (const [assetGroupName, assets] of Object.entries(groupedAssets)) {
 
   const isHermes = assets.find(asset => asset.endsWith('.hbc'));
   const windowsCallback = process.platform === 'win32' ? 'node ' : '';
-  execSync(`${windowsCallback}${sentryCliBin} sourcemaps upload ${isHermes ? '--debug-id-reference' : ''} ${assets.join(' ')}`, {
+  execSync(`${windowsCallback}${sentryCliBin} sourcemaps upload ${isHermes ? '--debug-id-reference' : ''} ${assets.join(' ')} --dist ${process.env.SENTRY_DIST}`, {
     env: {
       ...process.env,
       [SENTRY_PROJECT]: sentryProject,
